data/data/bootstrap/gcp/files/etc/NetworkManager/dispatcher.d/30-local-dns-prepender.template

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+IFACE=$1
+STATUS=$2
+case "$STATUS" in
```

Comment:
I am a bit confused by the switch here. Can we just move the major block out? The default case isn't doing anything here. Is there another intention ?

Reply:
I think the code to update resolv.conf does not need to be within the case statement. But, I am using what the on-prem platforms use and wanted to be consistent across all platforms that use in-cluster DNS. But, there is an ongoing effort to change the way resolv.conf is updated for all platforms that do this. This will get updated again at that time.